flows/inputs/msg.go

```diff
@@ -90,6 +94,7 @@ type msgInputEnvelope struct {
 	URN         urns.URN           `json:"urn" validate:"omitempty,urn"`
 	Text        string             `json:"text"`
 	Attachments []utils.Attachment `json:"attachments,omitempty"`
+	ExternalID  string             `json:"external_id,omitempty"`
```

Comment:
Should this be a `null.String` maybe? Just seems like differentiating null from empty may make sense in this case.

Reply:
We've thus far avoided nullable strings in the engine and I can't imagine a case where that needs to be differentiated in expressions.

Reply:
No reason to be afraid of nullable strings but fair point that expressions probably don't care about the difference between the two.